controllers/gce/controller/controller.go

```diff
@@ -307,15 +315,15 @@ func (lbc *LoadBalancerController) sync(key string) (err error) {
 
 	var syncError error
 	defer func() {
-		if deferErr := lbc.CloudClusterManager.GC(lbNames, nodePorts); deferErr != nil {
+		if deferErr := lbc.CloudClusterManager.GC(lbNames, allNodePorts); deferErr != nil {
 			err = fmt.Errorf("error during sync %v, error during GC %v", syncError, deferErr)
 		}
 		glog.V(3).Infof("Finished syncing %v", key)
 	}()
-
+	igs := []*compute.InstanceGroup{}
 	// Record any errors during sync and throw a single error at the end. This
 	// allows us to free up associated cloud resources ASAP.
-	if err := lbc.CloudClusterManager.Checkpoint(lbs, nodeNames, nodePorts); err != nil {
+	if igs, err = lbc.CloudClusterManager.Checkpoint(lbs, nodeNames, gceNodePorts, allNodePorts); err != nil {
```

Comment:
Since `igs` isn't being set anywhere else. I'd recommend killing the instantiation on ln 323 and breaking apart this line into two.